editor/components/block-list/style.scss

```diff
@@ -1,3 +1,71 @@
+.components-draggable__clone {
+	.editor-block-list__block-draggable {
+		background: white;
+		box-shadow: $shadow-popover;
+
+		@include break-small {
+			left: 0;
+			right: 0;
+		}
+	}
+
+	// Hide the Block UI when dragging the block
+	// This ensures the page scroll properly (no sticky elements)
+	.editor-block-contextual-toolbar,
+	.editor-block-mover,
+	.editor-block-settings-menu {
+		// I think important is fine here to avoid over complexing the selector
+		display: none !important;
+	}
+}
+
+.editor-block-list__block-draggable {
+	position: absolute;
+	top: 0;
+	right: 0;
+	bottom: 0;
+	left: 0;
+	z-index: z-index( '.editor-block-list__block-drag-inset' );
```

Comment:
Doesn't match the actual class name (there is no `.editor-block-list__block-drag-inset`).